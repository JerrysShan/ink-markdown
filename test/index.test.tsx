import Markdown from '../src/index';
import React from 'react';
import dedent from 'dedent';
import { render } from 'ink-testing-library';

test('Renders markdown', () => {
  const markdown = dedent`
    # Hello

    This is **markdown** printed in the \`terminal\`
  `;

  const { lastFrame } = render(<Markdown>{markdown}</Markdown>);

  expect(lastFrame()).toMatchSnapshot();
});
