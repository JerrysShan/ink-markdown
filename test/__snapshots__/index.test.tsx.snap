// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders markdown 1`] = `
"# Hello

This is markdown printed in the terminal"
`;
